Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'bootsnap', '>= 1.4.2', require: false
 gem 'jbuilder', '~> 2.7'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
+gem "rollbar", "3.3.0"
```

Comment:
I had to manually specify this version given that enabling `RollbarJS` in `config/initializers/rollbar.rb` was not working for versions `>= 3.3.1` – check [this issue](https://github.com/rollbar/rollbar-gem/issues/1100).